bundle/internal/schema/annotations_openapi_overrides.yml

```diff
@@ -230,6 +230,12 @@ github.com/databricks/cli/bundle/config/resources.Pipeline:
   "run_as":
     "description": |-
       PLACEHOLDER
+  "target":
+    # should not be marked as deprecated until its possible to switch
+    # existing pipelines to use 'schema' instead
+    "deprecated": false
```

Comment:
Is this set as `deprecated` in the annotations file already?

If not, absence of an entry means it won't be set.

Reply:
It's primarily for the case when parent file gets deprecated annotation from OpenAPI spec